packages/smooth_app/lib/helpers/analytics_helper.dart

```diff
@@ -2,13 +2,40 @@ import 'dart:async';
 
 import 'package:flutter/foundation.dart';
 import 'package:matomo_tracker/matomo_tracker.dart';
-import 'package:openfoodfacts/model/Product.dart';
-import 'package:openfoodfacts/utils/CountryHelper.dart';
-import 'package:openfoodfacts/utils/LanguageHelper.dart';
 import 'package:openfoodfacts/utils/OpenFoodAPIConfiguration.dart';
 import 'package:package_info_plus/package_info_plus.dart';
 import 'package:sentry_flutter/sentry_flutter.dart';
 
+enum AnalyticsCategory {
+  userManagement(tag: 'user management'),
+  scanning(tag: 'scanning'),
+  share(tag: 'share'),
+  couldNotFindProduct(tag: 'could not find product');
+
+  const AnalyticsCategory({required this.tag});
+  final String tag;
+}
+
+enum AnalyticsMessage {
```

Comment:
I would even call it `AnalyticsEvent`: would make more sense with the trackEvent calls.
And add a `///` comment.